SepulcherOfTheFirstOnes/Locales/zhCN.lua

```diff
@@ -4,7 +4,7 @@ if L then
 	L.traps = "陷阱" -- Stasis Trap
 	L.sparknova = "火花新星" -- Hyperlight Sparknova
 	L.relocation = "坦克炸弹" -- Glyph of Relocation
-	--L.relocation_count = "%s S%d (%d)" -- Tank Bomb S1 (1) // Tank Bomb (stage)(count)
+	L.relocation_count = "坦克炸弹:%d 之 %d" -- Tank Bomb S1 (1) // Tank Bomb (stage)(count)
```

Comment:
You can't just remove specific string like `%s` from translations. It will cause errors.